engine/crates/partial-caching/src/headers.rs

```diff
@@ -0,0 +1,25 @@
+pub struct RequestCacheControl {
+    /// Whether the cache code should use the cache to provide responses
+    ///
+    /// Controlled by the `no-cache` directive in headers
+    ///
+    /// The MDN description of `no-cache` seems to say that you can fetch
+    /// the cache but need to "revalidate" when this directive is present.
+    /// But we currently have no revalidation mechanism so this seems the
+    /// closest option
+    pub should_read_from_cache: bool,
```

Comment:
I would just keep the http header naming convention imho.

Reply:
I dunno - `no_cache` sort of implies "do absolutely no caching" to me.  Which is not what it actually means.  That and what we are treating the headers as meaning and what they originally mean are not _exactly_ the same thing.  Given both of those, using clearer more accurate names seemed good to me?

Reply:
hum, ok. fair point